docker/Dockerfile.rails.dev

```
@@ -0,0 +1,74 @@
+    && rm -rf /var/lib/apt/lists/*
+
+# Set working directory
+WORKDIR /app
+
+# Create non-root user for security
+RUN groupadd -g 1001 rails && \
+    useradd -m -u 1001 -g rails rails && \
+    chown -R rails:rails /app
+
+# Configure bundler for optimal development
+ENV BUNDLE_PATH="/usr/local/bundle" \
```

Hadolint (2.12.0)

[error] Do not refer to an environment variable within the same `ENV` statement where it is defined.

(DL3044)